cowprotocol/cowswap
# feat: add post-hook builder

# Summary

Adds post-hook builder. This will be handy for things like swap-n-bridge

<img width="695" alt="image" src="https://github.com/user-attachments/assets/92297704-f5e1-4021-8148-67ac468b3e50">

## Not in scope

This hook dapp is for demo porpoises and for DEVs only. It doesn't include the bells and whistles proposed in https://www.figma.com/file/1ctikx9sFa9Ff925GRsjvC/Hook-Store?type=design&node-id=7%3A116&mode=design&t=mohjmEtjEChWZFGJ-1

# To Test

Create a new post-hook using the builder
* 0x0B98057eA310F4d31F2a452B414647007d1645d9
* 28969
* 0xa3066aab00000000000000000000000079063d9173c09887d536924e2f6eadbabac099f5

Trade and make sure the hook is executed:
<img width="1404" alt="Screenshot at Aug 01 13-05-20" src="https://github.com/user-attachments/assets/517dd212-6373-4af8-bfa7-1f7afbddf799">
